src/app.js

```
@@ -1 +1,76 @@
+/* eslint-disable no-console */
 'use strict';
+
+import fs from 'fs';
+import path from 'path';
+
+function copyFile() {
+  if (process.argv.slice(2).length !== 2) {
+    console.error('length is less');
+
+    return;
+  }
+
+  const [entryFile, copyTo] = process.argv.slice(2);
+
+  if (!entryFile || !copyTo) {
+    console.error('One or two param is undefined');
+
+    return;
```

This undefined-parameter guard also only logs and returns. According to the checklist, missing/incorrect arguments must produce an error via `console.error` and then throw so the CLI fails loudly. Use a clear error message and throw after logging.